libraries/Wire/Wire.cpp

```diff
@@ -47,9 +44,9 @@ void TwoWire::_begin() {
 
     if (!is_master) {
         // Configure the read and write buffers for the I2C slave
-        w_status = cyhal_i2c_slave_config_read_buffer(&i2c_obj,  txBuffer, BUFFER_LENGTH);
+        w_status = cyhal_i2c_slave_config_read_buffer(&i2c_obj,  txBuffer._aucBuffer, BUFFER_LENGTH);
```

Comment:
Mmm, how is this working? 
The master always writes from the first position of the buffer.
And the slave feeds that buffer using the circular buffer interface?